recipes-multimedia/rpidistro-vlc/rpidistro-vlc_3.0.12.bb

```diff
@@ -0,0 +1,150 @@
+DESCRIPTION = "Video player and streamer - davinci edition"
+HOMEPAGE = "http://www.videolan.org"
+SECTION = "multimedia"
+
+LICENSE = "GPL-2.0"
+LIC_FILES_CHKSUM = "file://COPYING;md5=b234ee4d69f5fce4486a80fdaf4a4263"
+
+SRC_URI = "\
+  git://git@github.com/RPi-Distro/vlc;protocol=https;branch=buster-rpt \
+  file://0001-configure-fix-linking-on-RISC-V-ISA.patch \
+  file://0002-Revert-configure-Require-libmodplug-0.8.9.patch \
+  file://0003-mmal_20.patch \
+  file://0004-mmal_exit_fix.patch \
+  file://0005-mmal_chain.patch \
+  file://0006-Use-packageconfig-to-detect-mmal-support.patch \
+  file://0007-use-vorbisidec.patch \
+  file://0008-fix-luaL-checkint.patch \
+  file://0009-fix-EGL-macro-undeclared-and-EGLImageKHR.patch \
+  file://0010-fix-numeric_limits-is-not-a-member-of-std.patch \
+"
+
+SRCREV = "f7fd69f12a3b89d03768fa3bd468e8f33cd1dc7c"
+
+S = "${WORKDIR}/git"
+
+# While this item does not require it, it depends on ffmpeg which does
+LICENSE_FLAGS = "commercial"
```

Comment:
If so, Shouldn't ffmpeg have it?

Reply:
The rpidistro-ffmpeg recipe should also have it

Reply:
Sure. But should it also be added to all the recipes that depend on one that is commercially flagged? Why is that the case?